script/brave_license_helper.py

```diff
@@ -31,6 +31,11 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'vendor', 'brave-ios'),
         os.path.join('brave', 'vendor', 'brave_base'),
 
+        # These have auto-generated license files and
+        # GetThirdPartyDepsFromGNDepsOutput causes strange license errors
+        # unless the this entire directory is excluded.
+        os.path.join('brave', 'third_party', 'rust'),
```

Comment:
I think this can just exclude `//brave/third_party/rust/chromium_crates_io/vendor`.

Reply:
please see the comment explaining why the entire directory is excluded

Reply:
I read the comment, but I couldn't reproduce the errors. I suppose it doesn't matter much, but the narrower scope seemed a safer hack.

Reply:
I can try it again, but it failed with a very strange error before